src/app/routes/applications/applications.component.ts

```diff
@@ -65,6 +65,13 @@ export class ApplicationsComponent implements OnInit, AfterViewInit, OnDestroy {
   }
 
   openNewOrgComponent(): void {
+    if (!this.isIamEwcOwner) {
+      const namespace = 'orgowner.roles.orgcreator.apps.energyweb.iam.ewc';
+      const roleName = 'orgowner';
+      const siteUrl = `${location.origin}/#/enrol?org=${namespace}&roleName=${roleName}`;
```

Comment:
Url should look similar to `https://switchboard-dev.energyweb.org/#/enrol?roleName=orgowner&app=orgcreator.apps.testorg.iam.ewc`. This is because `orgcreator` is an `app` not an `org`
```suggestion
      const siteUrl = `${location.origin}/#/enrol?app=${namespace}&roleName=${roleName}`;
```